subsys/bluetooth/host/Kconfig

```diff
@@ -355,9 +355,20 @@ config BT_MAX_PAIRED
 	  default) number is 1.
 
 config BT_CREATE_CONN_TIMEOUT
-        int "Timeout for pending LE Create Connection command in seconds"
-        default 3
-        range 1 65535
+	int "Timeout for pending LE Create Connection command in seconds"
+	default 3
+	range 1 65535
+
+config BT_CONN_PARAM_UPDATE_TIMEOUT
+	int "Peripheral connection parameter update timeout in milliseconds"
+	default 5000
+	range 1 65535
+
+	help
+	  The value is a timeout used by slave device to wait until it
+	  starts connection parameters update procedure to change default
```

Comment:
the connection

Reply:
corrected

Reply:
I don't see this change yet?

Reply:
now it should be there, sorry for inconvenience